script/generate_licenses.py

```diff
@@ -161,36 +161,36 @@ def validated_data_field(data, field_name):
     return field_value
 
 
-def generate_backgrounds_license(preamble, backgrounds):
```

Comment:
we dont need to comment this out if already commented below

Reply:
actually yes, but this code is not needed by us